docs/build.js

```diff
@@ -49,6 +49,11 @@ async function run_with_theme(page, is_dark = false) {
     await page.goto("http://localhost:8080/");
     await page.setContent(template(is_dark));
     await page.setViewport(DEFAULT_VIEWPORT);
+    await page.evaluate(async () => {
+        while (!window.__TEST_PERSPECTIVE_READY__) {
```

Comment:
If `__TEST_PERSPECTIVE_READY__` were a `Promise`, we could just `await` it here.